plugins/components/health/Health.hpp

```diff
@@ -140,8 +140,12 @@ namespace Components {
                 std::cerr << "Warning: 'maxVal' not found in config. Using default value: 1.0f\n";
                 maxVal = 100;
             }
+
+            std::cout << "currentVal: " << currentVal << " maxVal: " << maxVal << std::endl;
+
```

Comment:
Leftover debug print ?

Reply:
This debug is here for the good continuity of the project! It will be removed when judged unnecessary! For now, its important to keep it to keep a trace on whats happening ^^

Reply:
Very well